test/functional/normalization/audit_logs_format_test.go

```diff
@@ -164,7 +164,7 @@ var _ = Describe("[Functional][LogForwarding][Normalization] message format test
 						Kind:             "Event",
 						Hostname:         framework.Pod.Spec.NodeName,
 						LogType:          "audit",
-						Level:            "default",
+						Level:            "Metadata",
```

Comment:
Why do we want this instead of some equivalent syslog severity?

Reply:
kube-apiserver events use the field "level" to mean something different - how much of the event has been deleted (request body, response body etc.) there's no syslog-style severity, and no way to derive it.